stack-lts-14.27.yaml

```diff
@@ -0,0 +1,4 @@
+resolver: lts-14.27
+
+# Fix weeder with stack-2.0
+# ghc-options: { "$locals": -ddump-to-file -ddump-hi }
```

Comment:
I commented out this line because I was seeing something weird on my machine:

```console
$ stack --version
Version 2.3.1, Git revision de2a7b694f07de7e6cf17f8c92338c16286b2878 (8103 commits) x86_64 hpack-0.33.0
$ stack --resolver stack-lts-14.27.yaml build
Exception while reading snapshot from stack-lts-14.27.yaml:
Couldn't parse snapshot from stack-lts-14.27.yaml: Error in $['ghc-options']['$locals']: failed to parse field 'ghc-options': Invalid package name: $locals
$ stack build
... (builds normally) ...
```

You can see when I do a `stack build` with `stack-lts-14.27.yaml`, I get the error `Couldn't parse snapshot from stack-lts-14.27.yaml: Error in $['ghc-options']['$locals']: failed to parse field 'ghc-options': Invalid package name: $locals`.  

I have no idea why this is happening.